systems/plants/urdf_interface/joint.cpp

```diff
@@ -197,7 +208,7 @@ bool JointSafety::initXml(TiXmlElement* config)
   const char* soft_lower_limit_str = config->Attribute("soft_lower_limit");
   if (soft_lower_limit_str == NULL)
   {
-    //ROS_DEBUG("joint safety: no soft_lower_limit, using default value");
+    ROS_DEBUG("joint safety: no soft_lower_limit, using default value");
```

Comment:
For some of the other attributes the debug message lists what the default value is. Do we want to do that throughout?